static/app/components/button.tsx

```diff
@@ -165,9 +184,41 @@ function BaseButton({
         return;
       }
 
+      const iconType = icon?.type.displayName;
```

Comment:
this was the way i thought of finding the displayName for the icons, but prettier thinks this is an error because type might not exist on icon. curious if there is a better way

Reply:
Yeah you would need to check if it's a react node first I think if you want to do this